File: src/main/java/upbrella/be/store/dto/response/AllImageUrlResponse.java
package upbrella.be.store.dto.response;

import lombok.Builder;
import lombok.Getter;

import java.util.List;

@Getter
@Builder
public class AllImageUrlResponse {

    private Long storeId;
    private List<SingleImageUrlResponse> images;

    public static AllImageUrlResponse of(Long storeId, List<SingleImageUrlResponse> images) {
        return AllImageUrlResponse.builder()
                .storeId(storeId)
                .images(images)
                .build();
    }
}

File: src/main/kotlin/upbrella/be/store/dto/response/AllImageUrlResponse.kt
package upbrella.be.store.dto.response

data class AllImageUrlResponse(
    val storeId: Long,
    val images: List<SingleImageUrlResponse>
) {
    companion object {
        fun of(storeId: Long, images: List<SingleImageUrlResponse>) =
            AllImageUrlResponse(storeId, images)
    }
}
